static/new_home/style.css

```diff
@@ -523,6 +522,12 @@ header
   }
 }
 
+@media screen and (max-width:600px) {
+  img {
```

Comment:
This changes the size of all the images on the page, whereas you just needed to change the size of images inside the about-us section.

Reply:
Oh okay